### ISSUE TITLE
ExpandableItemIndicator incorrect animation behavior on sample app

### ISSUE THREAD

**Issue by ibcurly** [h6ah4i/android-advancedrecyclerview#376]
The current behavior on the sample app Expandable+Draggable+Swipeable is as follows:
When a user clicks an item, the ExpandableItemIndicator is animated, and the item expands.
If that item is scrolled out of view so another item binds to its view holder, then the item is scrolled back into view, the expandable indicator is animated, when it should just change without the animation.
To recreate: expand the first item, scroll down to the bottom, then quickly scroll back up to the top and the expandable indicator will animate once you scroll to the top.

This behavior does not happen on the sample app Expandable (Basic), which this happens:
when the top item is expanded, then scrolled out of view, then scrolled back into view, the indicator is changed without animation. 

I believe the culprit has something to do with 
`final int expandState = holder.getExpandStateFlags();`
and
`            boolean animateIndicator = ((expandState & Expandable.STATE_FLAG_HAS_EXPANDED_STATE_CHANGED) != 0);`
in the adapter

**Comment by h6ah4i** [comment]
Hi. Thanks for the issue report 👍 

I have confirmed this is a BUG. Also, I think this is a library's one so the culprit is not being in the demo app code.

I have not found out the culprit yet, but I noticed that the `onViewRecycled()` method of the `ExpandableRecyclerViewWrapperAdapter` class is never invoked with the Expandable+Draggable+Swipeable demo. This is a totally unexpected behavior and it may causes some other severe bugs!

P.S. I am so busy working on my job, maybe the fix takes some weeks or more...
